### PR TITLE
CIV-10295 payment unsuccessful

### PR DESCRIPTION
**Pull request by kenneth-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-10295

### Change description ###
Set up payment unsuccessful

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/routes/features/claim/payment/claimFeePaymentUnsuccessfulController.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  saveDraftClaim,
+} from 'modules/draft-store/draftStoreService';
+import {getLng} from 'common/utils/languageToggleUtils';
+import {getFeePaymentRedirectInformation} from 'services/features/feePayment/feePaymentService';
+import {FeeType} from 'form/models/helpWithFees/feeType';
+const paymentUnsuccessfulController: Router = Router();
+
+const paymentUnsuccessfulViewPath  = 'features/claim/payment/claim-fee-payment-unsuccessful';
+
+async function renderView(res: Response, req: any, claim: Claim, claimId: string) {
```
Comment (manish14836) [inline src/main/routes/features/claim/payment/claimFeePaymentUnsuccessfulController.ts:22]: change request type(req: any) to AppRequest

File: src/main/routes/features/claim/payment/claimFeePaymentUnsuccessfulController.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      paymentUnsuccessfulBody: getPaymentUnsuccessfulBodyContent(claim, getLng(lang), claimId),
+    });
+}
+
+paymentUnsuccessfulController.get(PAY_CLAIM_FEE_UNSUCCESSFUL_URL, (async (req: AppRequest, res: Response, next: NextFunction) => {
+  try {
+    const claimId = req.params.id;
+    const paymentRedirectInformation = await getFeePaymentRedirectInformation(claimId, FeeType.CLAIMISSUED, req);
+    const claim = await getCaseDataFromStore(generateRedisKey(req));
+    claim.claimDetails.claimFeePayment=paymentRedirectInformation;
```
Comment (manish14836) [inline src/main/routes/features/claim/payment/claimFeePaymentUnsuccessfulController.ts:35]: spaces around =

File: src/main/routes/features/claim/payment/claimFeePaymentUnsuccessfulController.ts (hunk 1 above)
Comment (kenneth-hmcts) [reply src/main/routes/features/claim/payment/claimFeePaymentUnsuccessfulController.ts]: Updated from review

File: src/main/modules/i18n/locales/cy.json [hunk 3]
```diff
@@ -60,9 +60,11 @@
     "BUTTONS": {
       "ADD_ANOTHER_EVENT": "Ychwanegu digwyddiad arall",
       "BACK": "Yn ol",
+      "BACK_TO_CASE_DETAILS": "Back to case details",
       "CHANGE": "Newid",
       "CONTINUE": "Parhau",
       "CANCEL": "Canslo",
+      "GO_TO_ACCOUNT": "Go to your account",
```
Comment (GorilaNaranja) [inline src/main/modules/i18n/locales/cy.json:67]: Please search if the text is already translated and use the translation if we have it like this case.

"GO_TO_ACCOUNT": "Mynd i'ch cyfrif",

File: src/main/modules/i18n/locales/cy.json [hunk 4]
```diff
@@ -78,7 +80,8 @@
       "CLOSE_AND_RETURN_TO_CASE_OVERVIEW": "Cau a dychwelyd i drosolwg o'r achos"
     },
     "MICRO_TEXT": {
-      "HEARING_FEE": "Ffi gwrandawiad"
+      "HEARING_FEE": "Ffi gwrandawiad",
+      "CLAIM_FEE": "Claim fee"
```
Comment (GorilaNaranja) [inline src/main/modules/i18n/locales/cy.json:84]: Same here:
"CLAIM_FEE": "Ffi hawliad",

File: src/main/modules/i18n/locales/cy.json [hunk 5]
```diff
@@ -1357,7 +1360,7 @@
         "REQUESTED_MORE_TIME_TO_RESPOND": "Mae {{defendantName}} wedi gofyn am fwy o amser i ymateb.",
         "RESPONSE_BY_POST": "Bydd yr hawliad yn parhau drwy'r post",
         "WAITING_COURT_REVIEW": "Aros i'r llys adolygu'r achos",
-        "BOTH_SIGNED_SETTLEMENT_AGREEMENT": "You’ve both signed a settlement agreement.",       
+        "BOTH_SIGNED_SETTLEMENT_AGREEMENT": "You’ve both signed a settlement agreement.",
```
Comment (GorilaNaranja) [inline src/main/modules/i18n/locales/cy.json:1363]: We have this exact text other 3 times, and also we have the translation:
"SETTLEMENT_SIGNED": "Mae'r ddau ohonoch wedi llofnodi cytundeb setlo.",

File: src/main/modules/i18n/locales/cy.json [hunk 6]
```diff
@@ -2334,6 +2337,29 @@
       "PAGE_TITLE": "Partner with severe disability",
       "TITLE": "A oes gan eich partner anabledd difrifol?\n"
     },
+    "PAYMENT_CONFIRMATION": {
```
Comment (GorilaNaranja) [inline src/main/modules/i18n/locales/cy.json:2340]: Please review all the texts

File: src/main/modules/i18n/locales/cy.json (hunk 3 above)
Comment (kenneth-hmcts) [reply src/main/modules/i18n/locales/cy.json]: Updated from review

File: src/main/modules/i18n/locales/cy.json (hunk 4 above)
Comment (kenneth-hmcts) [reply src/main/modules/i18n/locales/cy.json]: Updated from review

File: src/main/modules/i18n/locales/cy.json (hunk 5 above)
Comment (kenneth-hmcts) [reply src/main/modules/i18n/locales/cy.json]: Updated from review

File: src/main/modules/i18n/locales/cy.json (hunk 6 above)
Comment (kenneth-hmcts) [reply src/main/modules/i18n/locales/cy.json]: Have a discussion on that, the related unsuccessful duplication translation is found already

**Review by manish14836 — APPROVED** [review]

**Review by GorilaNaranja — DISMISSED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Pull request merged**